core/embed/rust/src/ui/model_mercury/component/frame.rs

```diff
@@ -347,9 +355,16 @@ fn frame_event(
     header.event(ctx, event)
 }
 
-fn frame_place(header: &mut Header, footer: &mut Option<Footer>, bounds: Rect) -> Rect {
+fn frame_place(
+    header: &mut Header,
+    footer: &mut Option<Footer>,
+    bounds: Rect,
+    margin: usize,
+) -> Rect {
     let (mut header_area, mut content_area) = bounds.split_top(TITLE_HEIGHT);
-    content_area = content_area.inset(Insets::top(theme::SPACING));
+    content_area = content_area
+        .inset(Insets::top(theme::SPACING))
+        .inset(Insets::top(margin as i16));
```

Comment:
This makes the *real* margin now `SPACING + margin`. It's quite confusing but probably fine.

Reply:
Right. What could I do about it? Rename my parameter to `extra_margin`? Would that make more sense?